migration/sql-files/068-spaces_identities_fk.sql

```diff
@@ -0,0 +1,2 @@
+-- add a foreign key from spaces.owner_id to identities.id
+alter table spaces add constraint spaces_owner_id_identities_id_fk foreign key (owner_id) REFERENCES identities (id);
```

Comment:
I guess this will be removed for now due to the odd system.space migration hack. At that point the migration file has a bit of an odd name :)